tutorials/roofit/rf614_binned_fit_problems.C

```diff
@@ -82,14 +82,14 @@ void rf614_binned_fit_problems()
    // result and the true parameters comes from binning effects.
    std::unique_ptr<RooAbsData> expoData{generateBinnedAsimov(expo, x, 10000)};
 
-   // If you do the fit the usual was in RooFit, you will get a bias in the
+   // If you do the fit the usual way in RooFit, you will get a biased
    // result. This is because the continuous, normalized pdf is evaluated only
    // at the bin centers.
    std::unique_ptr<RooFitResult> fit1{expo.fitTo(*expoData, Save(), PrintLevel(-1), SumW2Error(false))};
    fit1->Print();
 
    // In the case of an exponential function, the bias that you get by
-   // evaluating the pdf only at the bin centers is a constant scale factor in
+   // evaluating the pdf only at the bin centers is an almost constant scale factor in
```

Comment:
Thanks for these improvements!

I just wonder why you want to say "almost" here? I think it's precisely constant: the integral of `exp(c*x)` is `1/c * exp(c*x)`, so as a definite integral from bin boundaries a to b that would be `1/c * (exp(c*b) - exp(c*a))`. If you divide this by the evaluation at the bin center, `exp(c*(a+b)/2)`, you get a factor that only depends only on `c` and the bin width `b - a`, which is usually constant.

So maybe it would be better to note "is a constant scale factor in each bin (in the case of uniform bins only)". But otherwise, I don't see where the "almost" is coming from. What am I missing?

Reply:
Sorry, I left the commit in the branch, but it wasn't to be pushed into the CMake Stuff. I'll open another PR with the RF-related things soon.

Reply:
Thank you!